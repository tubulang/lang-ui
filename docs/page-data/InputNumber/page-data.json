{"componentChunkName":"component---src-components-input-number-index-mdx","path":"/InputNumber","result":{"pageContext":{"frontmatter":{"name":"InputNumber 数字输入框","route":"/InputNumber","menu":"组件"},"entry":{"id":"881e1edc7cf6b95bec444aa8ac1cbfd1","filepath":"src/components/input_number/index.mdx","fullpath":"/Users/tubulang/personalProject/tubulang-ui/src/components/input_number/index.mdx","route":"/InputNumber","slug":"src-components-input-number-index","name":"InputNumber 数字输入框","menu":"组件","headings":[{"slug":"inputnumber-数字输入框","depth":1,"value":"InputNumber 数字输入框"},{"slug":"代码演示","depth":2,"value":"代码演示"},{"slug":"受控","depth":4,"value":"受控"},{"slug":"非受控","depth":4,"value":"非受控"},{"slug":"api","depth":2,"value":"API"}]}}},"staticQueryHashes":["1635659820"]}