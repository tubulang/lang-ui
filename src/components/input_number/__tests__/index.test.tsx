import React from 'react'
import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import InputNumber from '../input_number'

describe('InputNumber', () => {
  it('renders without error', () => {
    const { container } = render(<InputNumber defaultValue={4} />)
    expect(container).toMatchSnapshot()
  })
})
