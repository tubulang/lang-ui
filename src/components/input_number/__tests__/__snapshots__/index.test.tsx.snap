// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputNumber renders without error 1`] = `
<div>
  <input
    type="text"
    value=""
  />
</div>
`;
